src/amo/actions/reviews.js

```diff
@@ -268,3 +271,58 @@ export const hideReplyToReviewForm = (
 ): HideReplyToReviewFormAction => {
   return reviewIdAction({ type: HIDE_REPLY_TO_REVIEW_FORM, reviewId });
 };
+
+type FlagReviewParams = {|
+  errorHandlerId: string,
+  note?: string,
+  reason: FlagReviewReasonType,
+  reviewId: number,
+|};
+
+export type FlagReviewAction = {|
+  type: typeof SEND_REVIEW_FLAG,
+  payload: FlagReviewParams,
+|};
+
+export const flagReview = (
+  { errorHandlerId, note, reason, reviewId }: FlagReviewParams = {}
+): FlagReviewAction => {
+  if (!errorHandlerId) {
+    throw new Error('The errorHandlerId parameter is required');
+  }
+  if (!reason) {
+    throw new Error('The reason parameter is required');
+  }
+  if (!reviewId) {
+    throw new Error('The reviewId parameter is required');
```

Comment:
All of the "the"s in here seem redundant and are unlike the usual missing param error messages we have, so they can probably be removed.

Reply:
I think making it a complete sentence will help someone reading the exception message to parse it quicker and act on it quicker. I don't request that change in other people's code because it's not so important--it's just a personal preference that I think makes it easier for other developers.

Reply:
I'm fine with either, mostly I think we should just be consistent 😄